intellij_platform_sdk/BUILD

```diff
@@ -200,8 +200,8 @@ java_library(
 java_library(
     name = "jsr305",
     exports = select_for_ide(
-        clion = ["@jsr305_annotations//jar"],
-        default = [":plugin_api"],
+        default = ["@jsr305_annotations//jar"],
```

Comment:
With 2018.3 it appears jsr305.jar is not longer bundled with the IJ distribution. Or maybe the name changed.

Reply:
The built plugin seems to be working so far, jsr305 is only a compile time annotation right?